src/redux/greetings/greetingsSlice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const initialState = {
+  greeting: '',
+  isLoading: false,
+};
+
+const Api = 'http://127.0.0.1:3000/';
+
+export const fetchRandomGreeting = createAsyncThunk('greetings/fetchRandomGreeting', async () => {
+  const response = await fetch(Api);
+  const data = await response.json();
+  // console.log(data);
```

Comment:
- [ ] Kindly make sure to remove every commented code from your code base before pushing it to GitHub, it makes your code messy and cluttered

Reply:
I have removed the commented Code.
Thanks for reviewing